pallet/account-migration/src/lib.rs

```diff
@@ -173,8 +185,157 @@ pub mod pallet {
 		) -> DispatchResult {
 			ensure_none(origin)?;
 
+			Self::migrate_inner(from, to)?;
+
+			Ok(())
+		}
+
+		/// Similar to `migrate` but for multisig accounts.
+		///
+		/// The `_signature` should be provided by `who`.
+		#[pallet::call_index(1)]
+		#[pallet::weight(0)]
+		pub fn migrate_multisig(
+			origin: OriginFor<T>,
+			who: AccountId32,
+			others: Vec<AccountId32>,
+			threshold: u16,
+			to: AccountId20,
+			_signature: Signature,
+		) -> DispatchResult {
+			ensure_none(origin)?;
+
+			let (members, multisig) = multisig_of(who, others, threshold);
+
+			if threshold < 2 {
+				Self::migrate_inner(multisig, to)?;
+			} else {
+				let mut members = members.into_iter().map(|m| (m, false)).collect::<Vec<_>>();
+
+				// Set the status to `true`.
+				//
+				// Because the `_signature` was already been verified in `pre_dispatch`.
+				members
+					.last_mut()
```

Comment:
I guess you want to get `who` from members by using `last_mut()`?

If so, I think it is incorrect. The members returned from `multisig_of(xxx)` is sorted.

Reply:
Yep. Because I refactored some of them.